Resources/views/Collector/phpcr.html.twig

```diff
@@ -58,19 +87,36 @@
                 {% for i, call in calls %}
                     <tr id="queryNo-{{ i }}-{{ loop.parent.loop.index }}" class="{{ cycle(['odd', 'even'], i) }}">
                         <td>{{ loop.index }}</td>
-                        <td>{{ '%0.2f'|format(call.executionMS * 1000) }}&nbsp;ms</td>
+                        <td class="nowrap">{{ '%0.2f'|format(call.executionMS * 1000) }}&nbsp;ms</td>
                         <td>
-                            <code id="code-{{ i }}-{{ loop.parent.loop.index }}">
-                                {{ call.method|raw }}
-                            </code>
-                            <small>
-                                <strong>Method</strong>: {{ call.method|yaml_encode }} <br />
-                                <strong>Parameters</strong>: {{ call.params|yaml_encode }} <br />
-                                <strong>Environment</strong>: {{ call.env|yaml_encode }} <br />
+                            {% if profiler_markup_version == 1 %}
```

Comment:
do we need 2 different markup here ? DoctrineBundle manages to keep the same markup for both versions for its list of queries (except for some JS event listener needed for the old markup)